code/index.html

```diff
@@ -1,30 +1,148 @@
+                </div>
+            </div>
+            <div class="hero-image-details">
+                <div class="hero-image-text"> 
+                    <span>
+                        <h1>Even house plants need a holiday</h1>            
+                        <h3>Book your houseplants in with us whilst you're away</h3>
+                    </span>
+                </div>
+                <div>
+                    <button>DISCOVER</button>
+                </div>
+            </div>
+        </div>
+    </header>
 
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
+    <!-- MAIN SECTION WITH FORM & ACCORDIAN -->
+    <main>
+        <!-- FORM -->
+        <section class="form-parent">        
+            <form action="http://httpbin.org/anything" method="POST">
+                <div class="input-section">
+                    <h3>Subscribe to the house plant hotel newsletter</h3>
+                    <label for="fname">
+                        <input type="text" id="fname" name="fname" placeholder="First name" required>
+                    </label>
+                    <label for="lname">
+                        <input type="text" id="lname" name="lname" placeholder="Last name" required>
+                    </label>
+                    <label for="email">
+                        <input type="email" id="email" name="email" placeholder="Email" required>
+                    </label>
+                </div>
+                
+                <!-- RADIO BUTTON SECTION -->
+                <div class="radio-button-column">
+                    <h3>Interested in events at the plant house hotel?</h3>
+                    <div class="radio-button-row">
+                        <label class="radio-button-container">
+                            <input type="radio" id="yes" name="answer" checked required> 
+                            Yes please
+                            <span class="custom-radio"></span>
+                        </label>
 
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
-      </div>
+                        <label class="radio-button-container">
+                            <input type="radio" id="no" name="answer" required>  
+                            No thanks
+                            <span class="custom-radio"></span>
+                        </label>
+                    </div> 
+                </div>
+            
+                <!-- CHECKBOX SECTION -->
+                <div class="checkbox-column">
+                    <div class= "checkbox-row">
+                    <label class="checkbox-container">
+                        <input type="checkbox" id="terms" name="terms" checked required>
+                        I agree to terms & conditions
+                        <span class="custom-checkbox"></span>
+                    </label>
+                    </div>
+                </div>
```

Comment:
Your prefix approach is good `checkbox-` and `radio-`  This makes it very easy for the reader to understand the connected sections.